cli/integration/tests/waiter/cli.py

```diff
@@ -157,3 +152,28 @@ def show_token(waiter_url=None, token_name=None, flags=None):
 def output(cp):
     """Returns a string containing the stdout and stderr from the given CompletedProcess"""
     return f'\nstdout:\n{stdout(cp)}\n\nstderr:\n{decode(cp.stderr)}'
+
+
+def plugins_config():
+    """
+    If the WAITER_TEST_PLUGIN_JSON environment variable is set,
+    returns the parsed contents of the file, otherwise empty dict
+    """
+    if 'WAITER_TEST_PLUGIN_JSON' in os.environ:
+        path = os.environ['WAITER_TEST_PLUGIN_JSON']
+        content = util.load_json_file(os.path.abspath(path))
+        return content
+    else:
+        return {}
+
+
+def base_config():
+    """Returns a "base" config map that can be added to."""
+    return plugins_config()
+
+
+def write_base_config():
+    """Creates a config file that can be used as a starting point for integration tests"""
+    config = base_config()
+    if config:
+        write_json(os.path.abspath('.waiter.json'), config)
```

Comment:
Is it a problem from integration tests if this file is missing?

Reply:
No, it's only needed in some environments.